srcpkgs/jump/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'jump'
+pkgname=jump
+version=0.30.1
+revision=1
+build_style=go
+go_import_path=github.com/gsamokovarov/jump
+go_package=github.com/gsamokovarov/jump/cmd/jump
```

Comment:
Perhaps part of the problem is this doesn't exist.

Try removing go_package and see what happens?

Reply:
@Vaelatern That did the trick! I've marked this PR as ready for review.